pangeo_forge_recipes/writers.py

```diff
@@ -140,9 +141,9 @@ def write_combined_reference(
         raise NotImplementedError(f"{file_ext = } not supported.")
     return ReferenceFileSystem(
         outpath,
-        target_options=target_options,
-        remote_options=remote_options,
-        remote_protocol=remote_protocol,
+        target_options=full_target.get_fsspec_credential_kwargs(),
+        remote_options=full_target.get_fsspec_credential_kwargs(),
+        remote_protocol=full_target.get_fsspec_remote_protocol(),
```

Comment:
so the general idea is nested `fsspec` class instantiation like this instead of doing something gross like unpacking auth creds from an `FSSpec` instance we just use `config.Config.target_storage` from https://github.com/pangeo-forge/pangeo-forge-recipes/pull/652